Cargo.toml

```diff
@@ -52,6 +52,7 @@ semver = { version = "1.0.3", features = ["serde"] }
 serde = { version = "1.0.123", features = ["derive"] }
 serde_ignored = "0.1.0"
 serde_json = { version = "1.0.30", features = ["raw_value"] }
+serde-value = "0.7.0"
```

Comment:
I am worried about adding this as a dependency since it hasn't been updated since [July 15th, 2020](https://github.com/arcnmx/serde-value) and the comment you linked to about `serde-value` appears to be [6 years old](https://github.com/serde-rs/serde/blame/f52d134c14f6904010d0a59107987b050b36b811/serde/src/private/de.rs#L218)

Someone on the [Cargo team](https://www.rust-lang.org/governance/teams/dev-tools#cargo) could have a different opinion so I would wait for them to comment as well.

Reply:
`serde-value` seems innocuous enough and we can always vendor it if needed.  I think how much it cleans up this code shows it pulls its weight.